HIP/hip-655.md

```diff
@@ -0,0 +1,106 @@
+
+## Rationale
+
+Token airdrops are very common in the blockchain industry and many legitimate use cases require airdropping of tokens. This HIP attempts to provide a mechanism to enable these airdrops while still protecting the account owners, and imposing dis-incentivizing costs to the senders of the airdropped tokens.
+
+## User Stories
+
+### Terminology
+
+The following terminology is used in the rest of this Specification
+
+- Airdrop-Sender: the owner of a token that they wish to send to somebody without the explicit association/permission from the intended recipient
+- Airdrop-Recipient: the intended recipient of the above airdrop
+- Node-Operator: the operator of a node in the Hedera network
+    
+### User stories
+    
+- As an Airdrop-Sender, I want to send arbitrary tokens (Fungible and/or Non-fungible) to Airdrop-Recipients so that I can create interesting business use cases
+    1. I understand that the recipients of the airdrop may not wish to accept these tokens and I am willing to pay for the network resources used while they make their decision.
+    2. If the intended Airdrop-Recipients do not accept my transfer, I want to be able to restore those tokens to my account so that I can do whatever I wish with those tokens.
+- As an Airdrop-Recipient, I want to receive arbitrary tokens (Fungible and/or Non-fungible) from the Airdrop-Senders so that I can accumulate those in my account.
+    1. I want to achieve the above while having explicit control over what tokens actually show up in my account.
+    2. I want to achieve the above without being subject to the reputational/legal/spam risk of being associated with unwanted tokens
+    3. I want to achieve the above without having to be forced to pay for the storage/renewal of these unwanted tokens unless I explicitly accept those.
+- As a node operator in the network, I want to make sure that the network security is not compromised and that somebody pays for the network resources that are used. The design of this HIP is partially inspired by how we handle spam emails in today's world, but with an added safeguard of ensuring that there is an economic disincentive for an Airdrop-Sender from sending indiscriminate spam. Through this HIP, we will create a new concept of unsolicited-token-outbox/inbox that maintain a record of intended transfers of airdrops.
+    
+## Specification
```

Comment:
We should add product specification sections to show changes to proto, services, SDK, Mirror Node, mirror node explorer and relay. This HIP actually touches all of them :)

Reply:
Maybe we should split it up into several hips?

Reply:
One HIP with a specification section for each product since it's a single feature